python/lsst/obs/base/defineVisits.py

```diff
@@ -890,10 +908,12 @@ class _GroupExposuresByGroupMetadataConfig(GroupExposuresConfig):
 
 @registerConfigurable("by-group-metadata", GroupExposuresTask.registry)
 class _GroupExposuresByGroupMetadataTask(GroupExposuresTask, metaclass=ABCMeta):
-    """An exposure grouping algorithm that uses exposure.group_name and
-    exposure.group_id.
+    """An exposure grouping algorithm that uses the exposure group.
+
+    This algorithm uses the ``group`` dimension for modern universes and the
+    ``exposure.group_id`` for older universes.
 
-    This algorithm _assumes_ exposure.group_id (generally populated from
+    This algorithm _assumes_ group ID (generally populated from
```

Comment:
```suggestion
    This algorithm *assumes* group ID (generally populated from
```
Haven't tested this particular case, but my recollection is that rST does not like underscores for emphasis; it's either single or double asterisks.